addons/website/static/src/client_actions/configurator/configurator.js

```diff
@@ -302,19 +302,19 @@ class PaletteSelectionScreen extends Component {
         const logoSelectInput = this.logoInputRef.el;
         if (logoSelectInput.files.length === 1) {
             const file = logoSelectInput.files[0];
-            const data = await utils.getDataURLFromFile(file);
-            const attachment = await this.rpc('/web_editor/attachment/add_data', {
-                'name': 'logo',
-                'data': data.split(',')[1],
-                'is_image': true,
-            });
-            if (!attachment.error) {
+            try {
+                const data = await utils.getDataURLFromFile(file);
+                const attachment = await this.rpc('/web_editor/attachment/add_data', {
+                    'name': 'logo',
+                    'data': data.split(',')[1],
+                    'is_image': true,
+                });
```

Comment:
Same question.
Do you have an use case for the fix in this file?

Reply:
As here also the same URL '/web_editor/attachment/add_data' is called, so while uploading logo this error might occur, that's why putted 'data' in try-catch block here.